configs/problem/ic_ond18_reaction.yaml

```diff
@@ -0,0 +1,64 @@
+# @package _group_
+protocol: 'ond'
+workdir: ''
+harness: 'local'
+domain: "image_classification"
+test_ids:
+- "OND.54011215.0000.1236"
+hints:
+- "red_light"
+use_feedback: True
+is_eval_enabled: True
+is_eval_roundwise_enabled: True
+dataset_root: ""
+detectors:
+  has_reaction_baseline: True
+  has_baseline: False
+  csv_folder: image_classification
+  baseline_class: "BaselineImageClassifier"
+  detector_configs:
+    BaselineImageClassifier:
+      model_name: "resnet50"
+      checkpoint_path: ""
+      image_size: 256
+      crop_size: 224
+      batch_size: 100
+      num_workers: 6
+      num_classes: 413
+      use_gpu: True
+    OND_18_With_Redlight:
+      cnn_params:
+          model_path: ""
+          known_classes: 413
+          known_unknown_classes: 51
+          image_size: 320
+      evm_params:
+         model_path: ""
+         tail_size: 40000
+         cover_threshold: 0.8
+         distance_multiplier: 0.65
+         distance_function: "cosine"
+         chunk_size: 200
+         gpu_list:
```

Comment:
multi gpu option removed. now it work on only in single gpu

Reply:
Addressed in a8d4a29